torch/distributed/tensor/parallel/style.py

```diff
@@ -563,6 +563,34 @@ def __init__(
         output_layouts=Replicate(),
```

Comment:
I think it's also worth updating `RowwiseParallel` and `ColwiseParallel` doc to explicitly say what modules we support:

ColwiseParallel: nn.Linear, nn.Embedding
RowwiseParallel: nn.Linear

Reply:
sure, I can add that here.